examples/gaze-detection/gaze.py

```diff
@@ -6,10 +6,10 @@
 import os
 
 IMG_PATH = "image.jpg"
-API_KEY = os.environ["API_KEY"]
+ROBOFLOW_API_KEY = os.environ["ROBOFLOW_API_KEY"]
```

Comment:
This value is fine, as we also allow it in other places as an alternative for `API_KEY` - in long ran we may decide on single one

Reply:
Let's use `ROBOFLOW_API_KEY` across docs. Good note that the API_KEY = os.environ["API_KEY"]
here needs to be updated: https://inference.roboflow.com/foundation/gaze/#how-to-use-l2cs-net